v3/src/components/graph/components/graph.tsx

```diff
@@ -34,14 +34,12 @@ interface IProps {
   graphRef: MutableRefObject<HTMLDivElement>
 }
 
-const marqueeState = new MarqueeState()
-
-
 export const Graph = observer(function Graph({graphController, graphRef}: IProps) {
   const graphModel = useGraphModelContext(),
     { enableAnimation, dotsRef } = graphController,
     {plotType} = graphModel,
     instanceId = useInstanceIdContext(),
+    marqueeState = useMemo<MarqueeState>(() => new MarqueeState(), []),
```

Comment:
This is fine, but just to point out another option, it could also be a `volatile` property on the model. If we were ever to support a view-in-window type of functionality, that might have some advantage.

Reply:
Interesting. I didn't think of making it a model property at all.